docker/maistra-builder_1.2.Dockerfile

```diff
@@ -6,6 +6,9 @@ ENV KUBECTL_VERSION="v1.17.0"
 ENV HELM_VERSION="v2.16.6"
 ENV VALE_VERSION="v2.1.1"
 
+#this needs to match the version of Hugo used in Maistra.io's netlify.toml file
+ENV HUGO_VERSION="0.69.2"
```

Comment:
Does it make sense to use the latest one (0.70)? Does netlify support it?

Reply:
Netlify supports any Hugo version. Fedora 32 has support for 0.69. I'm not sure what Fedora 31 and RHEL have available. But developers could manually install this way too.

Reply:
OK, either way works for me